src/progress-bar/internal.tsx

```diff
@@ -11,39 +11,77 @@ import InternalStatusIndicator from '../status-indicator/internal';
 import { ProgressBarProps } from './interfaces';
 import styles from './styles.css.js';
 
+import { throttle } from '../internal/utils/throttle';
+import LiveRegion from '../internal/components/live-region';
+
 const MAX_VALUE = 100;
+const ASSERTION_FREQUENCY = 5000; // interval in ms between progress announcements
 
 const clamp = (value: number, lowerLimit: number, upperLimit: number) => {
   return Math.max(Math.min(value, upperLimit), lowerLimit);
 };
 
 interface ProgressProps {
+  label: ReactNode;
+  type: 'percentage' | 'ratio';
   value: number;
+  maxValue: number;
   isInFlash: boolean;
   labelId: string;
+  ariaValueText?: string;
 }
-export const Progress = ({ value, isInFlash, labelId }: ProgressProps) => {
+export const Progress = ({
+  label,
+  value,
+  maxValue = MAX_VALUE,
+  type,
+  isInFlash,
+  labelId,
+  ariaValueText,
+}: ProgressProps) => {
   const roundedValue = Math.round(value);
-  const progressValue = clamp(roundedValue, 0, MAX_VALUE);
+  const progressValue = clamp(roundedValue, 0, maxValue);
+  const isRatio = type === 'ratio';
+  const percentage = isRatio && maxValue !== 100 ? Math.round(((progressValue * 1.0) / maxValue) * 100) : progressValue;
+  const valueText = isRatio ? ariaValueText || `${progressValue}/${maxValue}` : `${percentage}%`;
+
+  const [assertion, setAssertion] = useState('');
+  const throttledAssertion = useMemo(() => {
+    return throttle((value: ProgressBarProps['value']) => {
+      const announcement = type === 'ratio' ? `${value} of ${maxValue}}` : `${value}%`;
+      setAssertion(`${label ?? ''}: ${announcement}`);
+    }, ASSERTION_FREQUENCY);
+  }, [label, type, maxValue]);
+
+  useEffect(() => {
+    throttledAssertion(value);
+  }, [throttledAssertion, value]);
 
   return (
-    <div className={styles['progress-container']}>
-      <progress
-        className={clsx(
-          styles.progress,
-          progressValue >= MAX_VALUE && styles.complete,
-          isInFlash && styles['progress-in-flash']
-        )}
-        max={MAX_VALUE}
-        value={progressValue}
-        aria-labelledby={labelId}
-      />
-      <span aria-hidden="true" className={styles['percentage-container']}>
-        <InternalBox className={styles.percentage} variant="small" color={isInFlash ? 'inherit' : undefined}>
-          {`${progressValue}%`}
-        </InternalBox>
-      </span>
-    </div>
+    <>
+      <div className={styles['progress-container']}>
+        <progress
+          className={clsx(
+            styles.progress,
+            progressValue >= maxValue && styles.complete,
+            isInFlash && styles['progress-in-flash']
+          )}
+          max={maxValue}
+          value={progressValue}
+          aria-valuemin={0}
+          aria-valuemax={maxValue}
```

Comment:
We've removed these in the past for the regular percentage type (https://github.com/cloudscape-design/components/pull/1082). I would suggest that we only add the new aria props for the new ratio type, and keep the old percentage props unchanged.

But also, let's first check if `aria-valuemin`, `aria-valuemax`, and `aria-valuenow` are even needed for the ratio type. Maybe `min`, `max` and `value` are sufficient here as well.